### PR TITLE
commit 1508 from pc bit_operations + hwork with arrays2

### PR DESCRIPTION
**Pull request by VladimirKhinoy**

### CODE REVIEW

File: SensorAverage/Vladimir/SensorAverage.cs [hunk 1]
```diff
@@ -9,14 +9,13 @@ namespace SensorAverage.Vladimir
     class SensorAverage : ISensorsAverage
     {
 
-        int countControllers = 8;    // максимум №№ конроллеров на трех разрядах
-        int countBlocs = 10;         // число получаемых блоков для экперимента
+        int countControllers = 8;    // максимум №№ контроллеров на трех разрядах
+        //int countBlocs;         // число получаемых блоков для эксперимента  //int countBlocs = 10
```
Comment (jane-u) [inline SensorAverage/Vladimir/SensorAverage.cs]: please remove unnecessary comments

File: SensorAverage/Vladimir/SensorAverage.cs [hunk 2]
```diff
@@ -26,8 +25,8 @@ class SensorAverage : ISensorsAverage
                   }
               }
 
-              if (countTrueBlocs != 0)
-              {
+              //if (countTrueBlocs != 0)
```
Comment (jane-u) [inline SensorAverage/Vladimir/SensorAverage.cs]: please remove unnecessary comments

File: SensorAverage/Vladimir/SensorAverage.cs [hunk 3]
```diff
@@ -43,22 +42,38 @@ class SensorAverage : ISensorsAverage
...
+                  {
+                          if (dataOut[i].value != 0)
+                          {
+                              answerTuple[indexWithoutZerro] = Tuple.Create(dataOut[i].code, dataOut[i].value);
+                              Console.WriteLine(" Проверка: Код = {0} значение = {1}", Convert.ToString(answerTuple[indexWithoutZerro].Item1,2),answerTuple[indexWithoutZerro].Item2);
+                              indexWithoutZerro++;
+                          }
+                  }
+                
+              //}
```
Comment (jane-u) [inline SensorAverage/Vladimir/SensorAverage.cs]: please remove unnecessary comments

**Comment by VladimirKhinoy** [comment]
work by bit & arrays without bad of comments

**Pull request merged**